opcua/server/internal_subscription.py

```diff
@@ -140,10 +142,12 @@ def _delete_monitored_items(self, mid):
 
     def datachange_callback(self, handle, value, error=None):
         if error:
-            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'", self, handle, error)
+            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'",
```

Comment:
But 80 characters length according to PEP8 I find not soo cool. I would than rather agree with 120 or something like that.